Makefile

```diff
@@ -13,8 +13,7 @@ GOVUK_ROOT_DIR ?= "${HOME}/govuk"
 APPS ?= $(shell ls services/*/Makefile | xargs -L 1 dirname)
 
 default:
-	@echo "Run 'make build' to bootstrap govuk-docker"
-	@echo "Or 'make APP-NAME' to set up an app"
+	@echo "Run 'make APP-NAME' to set up an app and its dependencies"
```

Comment:
Should we put an example ie. `make content-publisher` or is it obvious to first timers?

Reply:
Sure, I can add an example